### PR TITLE
[Firmware ] Run SONIC command

### PR DESCRIPTION
**Pull request by KostiantynYarovyiBf**
#### What I did
add log Not supported firmware updates if platform is barefoot

#### How I did it
add raise exception if platform is barefoot

#### How to verify it
Run show platform firmware updates

### CODE REVIEW

File: fwutil/main.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
     from .lib import PlatformDataProvider, ComponentStatusProvider, ComponentUpdateProvider
     from .lib import URL, SquashFs
     from .log import LogHelper
+    from sonic_py_common import device_info
```
Comment (novikauanton) [inline fwutil/main.py:15]: To simplify dependencies management imports should be grouped in next order:
```
1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
```

https://www.python.org/dev/peps/pep-0008/#imports

File: fwutil/main.py [hunk 2]
```diff
@@ -348,6 +349,13 @@ def show():
 def updates(ctx, image):
     """Show available updates"""
     try:
+        # Barefoot
+        if(str(device_info.get_platform()) == "x86_64-accton_as9516bf_32d-r0"   or
```
Comment (novikauanton) [inline fwutil/main.py:353]: the expression can be simplified 
```
platforms = {
  "x86_64-accton_as9516bf_32d-r0",
  "x86_64-accton_as9516_32d-r0",
  "x86_64-accton_wedge100bf_32x-r0",
  "x86_64-accton_wedge100bf_65x-r0",
}
if device_info.get_platform() in platforms :
    ...
```

File: fwutil/main.py (hunk 1 above)
Comment (dmytroxshevchuk) [reply fwutil/main.py:15]: Interesting from https://www.python.org/dev/peps/pep-0008/#imports

```
# Correct:
import os
import sys
# Wrong:
import sys, os
```
Is it affect logic of work or its just a recommendation for style of code?

File: fwutil/main.py (hunk 2 above)
Comment (msosyak) [reply fwutil/main.py:353]: What if a new barefoot platform will be added?
Please use `get_platform_info` and check by `asic_type` filed. https://github.com/msosyak/sonic-buildimage/blob/2e7e7814cc71c353bef179c7f29bf00fcbac8714/src/sonic-py-common/sonic_py_common/device_info.py#L327

File: fwutil/main.py [hunk 3]
```diff
@@ -348,6 +349,13 @@ def show():
...
+           str(device_info.get_platform()) == "x86_64-accton_as9516_32d-r0"     or
+           str(device_info.get_platform()) == "x86_64-accton_wedge100bf_32x-r0" or
+           str(device_info.get_platform()) == "x86_64-accton_wedge100bf_65x-r0" ):
+            raise NameError("Firmware updates are not supported for barefoot")
```
Comment (msosyak) [inline fwutil/main.py:357]:
```suggestion
            raise NameError("Firmware updates are not supported by barefoot devices")
```

File: fwutil/main.py (hunk 1 above)
Comment (KostiantynYarovyiBf) [reply fwutil/main.py]: done

File: fwutil/main.py (hunk 2 above)
Comment (KostiantynYarovyiBf) [reply fwutil/main.py]: done

File: fwutil/main.py (hunk 3 above)
Comment (KostiantynYarovyiBf) [reply fwutil/main.py]: done

**Review by msosyak — APPROVED** [review]

**Comment by KostiantynYarovyiBf** [comment]
@jleveque please review

**Comment by KostiantynYarovyiBf** [comment]
close this PR because add correct fix with another PR
https://github.com/Azure/sonic-buildimage/pull/8690

**Pull request closed without merging**